package.json

```diff
@@ -56,18 +55,17 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "fs-extra": "11.1.1",
     "husky": "8.0.3",
-    "jest": "29.7.0",
-    "jest-environment-jsdom": "29.7.0",
+    "jsdom": "^22.1.0",
     "lint-staged": "15.0.2",
     "npm-run-all": "4.1.5",
     "prettier": "3.0.3",
     "rollup": "4.3.0",
     "sass": "1.69.5",
-    "ts-jest": "29.1.1",
     "ts-loader": "9.5.0",
     "tslib": "2.6.2",
     "typescript": "5.2.2",
-    "vite": "3.2.7",
+    "vite": "4.5.0",
```

Comment:
You can't update this read more in https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/258#issuecomment-1794608629